RecoLocalTracker/SiPixelDigiReProducers/python/SiPixelDigiMorphed_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
the file should be renamed from `SiPixelDigiMorphed_cfi.py` to `siPixelDigisMorphed_cfi.py` to match the name of the module defined in it.
https://github.com/cms-sw/cmssw/pull/34606#discussion_r678742575